browser/tor/test/onion_location_navigation_throttle_browsertest.cc

```diff
@@ -57,6 +60,8 @@ std::unique_ptr<net::test_server::HttpResponse> HandleOnionLocation(
   http_response->set_content("<html><head></head></html>");
   if (request.GetURL().path_piece() == kTestOnionPath) {
     http_response->AddCustomHeader("onion-location", kTestOnionURL);
+    // Subsequent headers should be ignored.
```

Comment:
Don't we already ignore subsequent header without this PR? Or is this just extending test coverage?

Reply:
Just extending test coverage.